ansible/roles/ooni-backend/tasks/cron.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Template update bouncer script
+  template: >
+    src="update_bouncer.j2"
```

Comment:
In our current stage/testing this would be very difficult to automate if we use hard-coded values.
Since we are going to be needing two cronjobs --a testing and a staging one that have different configuration and files.